docs/firstboot-configuration.md

```diff
@@ -0,0 +1,187 @@
+# Firstboot OS Configuration
+
+This section provides an overview of how to configure the OS at firstboot.
+
+## The runtime context
+
+Unified Core images crafted by `elemental3ctl` include, by default, two partitions, an ESP partition
+(bootloader, kernel and initrd) and a Linux partition (the OS itself). The OS partition is a btrfs
+partition including several subvolumes where there is the default read-only subvolume mounted as the
+root device and a list of read-write subvolumes which are mounted in paths that are expected or required
+to be read-write, such as `/etc` or `/var`. The default subvolume is a btrfs read-only subvolume so
+regardless of mounting it with or without read-write capabilities, the filesystem will prevent any write
+in there causing a failure in any attempt.
+
+Default read-write subvolumes for `elemental3ctl` installations are:
+
+* /var
```

Comment:
would prefer for all of them to be in backticks to have a similar optical style to the rest of the text, e.g. `/var` and so on.